src/stores/ShardDetail.ts

```diff
@@ -85,74 +237,114 @@ export const getInitialState = (
                 'Shard List Set'
             );
         },
-        getShardStatusColor: (catalogNamespace) => {
-            const { shards } = get();
+        getTaskShards: (catalogNamespace, shards) => {
```

Comment:
`getTaskShards` returns all the shards associated with a specific task (i.e., that designated by the `catalogNamespace` input argument).